tools/StaticAnalysis/Exceptions/Az.Network/SignatureIssues.csv

```diff
@@ -372,3 +372,22 @@
+"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.NewAzureApplicationGatewaySslProfileCommand","New-AzApplicationGatewaySslProfile","1","8100","New-AzApplicationGatewaySslProfile Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
+"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.NewAzureApplicationGatewaySslProfileCommand","New-AzApplicationGatewaySslProfile","1","8410","Parameter TrustedClientCertificates of cmdlet New-AzApplicationGatewaySslProfile does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
+"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.RemoveAzureApplicationGatewaySslProfileCommand","Remove-AzApplicationGatewaySslProfile","1","8100","Remove-AzApplicationGatewaySslProfile Does not support ShouldProcess but the cmdlet verb Remove indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
+"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.SetAzureApplicationGatewaySslProfileCommand","Set-AzApplicationGatewaySslProfile","1","8100","Set-AzApplicationGatewaySslProfile Does not support ShouldProcess but the cmdlet verb Set indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
+"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.SetAzureApplicationGatewaySslProfileCommand","Set-AzApplicationGatewaySslProfile","1","8410","Parameter TrustedClientCertificates of cmdlet Set-AzApplicationGatewaySslProfile does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
+"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.SetAzureApplicationGatewaySslProfilePolicyCommand","Set-AzApplicationGatewaySslProfilePolicy","1","8410","Parameter DisabledSslProtocols of cmdlet Set-AzApplicationGatewaySslProfilePolicy does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
+"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.RemoveAzureApplicationGatewaySslProfilePolicyCommand","Remove-AzApplicationGatewaySslProfilePolicy","1","8100","Remove-AzApplicationGatewaySslProfilePolicy Does not support ShouldProcess but the cmdlet verb Remove indicates that it should.	Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
```

Comment:
This line and the followings were strangely formatted, causing static analysis tool failing to parse the file. Please fix them.

```suggestion
"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.RemoveAzureApplicationGatewaySslProfilePolicyCommand","Remove-AzApplicationGatewaySslProfilePolicy","1","8100","Remove-AzApplicationGatewaySslProfilePolicy Does not support ShouldProcess but the cmdlet verb Remove indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
```